### ISSUE TITLE
add command allows to add package multiple times.

### ISSUE THREAD

**Issue by rhalff** [shyndman/pubx#6]
Hi, first of all thanks for writing this package, it's quite useful.

I've mistakenly added a package multiple times and right now the tool doesn't protect me from that mistake.

The expected behavior for me would be to just ignore the addition of a package which already exists, without any warning or error.

**Comment by shyndman** [comment]
Hey @rhalff,

That sounds like a reasonable request. @Vanethos What do you think?

**Comment by shyndman** [comment]
Worth mentioning, the behavior of npm in this situation is to bump the version in package.json to latest.

Would that also be acceptable?

**Comment by rhalff** [comment]
Copying the behavior of npm would be preferred, so yes.

**Comment by shyndman** [comment]
Cool.

@Vanethos, let me know if you want me to jump in. I'm happy to take it if you're busy.

**Comment by Vanethos** [comment]
@shyndman I'll be busy for the next couple of days and then I'll have some time, if you want to release it ASAP, please do go ahead :D 

@rhalff Thanks for the headsup!

**Comment by shyndman** [comment]
Cool, I'll take it.

**Comment by shyndman** [comment]
@rhalff: Just published the new version, 1.1.0. Go ahead and `pub activate`. :)

**Comment by rhalff** [comment]
Thanks, work well.

I've only found one edge case, where it's allowed to not specify a version number:
https://dart.dev/tools/pub/dependencies

```yaml
$ grep build_runner pubspec.yaml
  build_runner:

$ pubx add build_runner --dev   
+ build_runner: ^1.6.5

$ grep build_runner pubspec.yaml
  build_runner^1.6.5:

$ pubx add build_runner --dev   
+ build_runner: ^1.6.5

$ grep build_runner pubspec.yaml
  build_runner: ^1.6.5
  build_runner^1.6.5:
```
That was a deliberate attempt to try something different and break it though.

**Comment by shyndman** [comment]
Thanks. Created a new issue.

**Comment by shyndman** [comment]
Fixed in 1.1.1, which is now published.
